### PR TITLE
Fix AirloopHVAC:UnitarySystem with variable speed coils and setpoint control

### PR DESCRIPTION
**Pull request by rraustad**
Addresses #5067 and #4861 
This fixes problems with AirloopHVAC:UnitarySystem with Coil:_:WaterToAirHeatPump:VariableSpeedEquationFit & Coil:_:DX:VariableSpeed when using Setpoint control.

### CODE REVIEW

**Comment by rraustad** [comment]
Example file UnitarySystem_VSHeatPumpWaterToAirEquationFit is the only diff. I would expect differences in this file given the changes here.

File: src/EnergyPlus/HVACUnitarySystem.cc [hunk 1]
```diff
@@ -8964,7 +8992,8 @@ namespace HVACUnitarySystem {
 
 				} else if ( ( SELECT_CASE_var == CoilDX_MultiSpeedHeating ) || ( SELECT_CASE_var == Coil_HeatingAirToAirVariableSpeed ) || ( SELECT_CASE_var == Coil_HeatingWaterToAirHPVSEquationFit ) || ( SELECT_CASE_var == Coil_HeatingElectric_MultiStage ) || ( SELECT_CASE_var == Coil_HeatingGas_MultiStage ) ) {
 
-					SimMultiSpeedCoils( UnitarySysNum, AirLoopNum, FirstHVACIteration, SensibleLoad, LatentLoad, PartLoadFrac, HeatingCoil );
+					UnitarySystem( UnitarySysNum ).HeatingCoilSensDemand = ReqOutput;
+					SimVariableSpeedCoils( "", UnitarySystem( UnitarySysNum ).HeatingCoilIndex, FanOpMode, UnitarySystem( UnitarySysNum ).MaxONOFFCyclesperHour, UnitarySystem( UnitarySysNum ).HPTimeConstant, UnitarySystem( UnitarySysNum ).FanDelayTime, CompOn, CycRatio, SpeedNum, SpeedRatio, SensLoad, dummy );
```
Comment (rraustad) [inline src/EnergyPlus/HVACUnitarySystem.cc:9009]: This looks wrong. SimVariableSpeedCoils should not be called for multispeed coils.

**Comment by mjwitte** [comment]
@rraustad If you can, please merge in current develop and resolve conflicts.

File: src/EnergyPlus/HVACUnitarySystem.cc [hunk 2]
```diff
@@ -7997,9 +8013,14 @@ namespace HVACUnitarySystem {
 
 					SimDXCoilMultiSpeed( CompName, 0.0, PartLoadFrac, UnitarySystem( UnitarySysNum ).CoolingCoilIndex );
 
-				} else if ( ( CoilType_Num == CoilDX_MultiSpeedCooling ) || ( CoilType_Num == Coil_CoolingAirToAirVariableSpeed ) || ( CoilType_Num == Coil_CoolingWaterToAirHPVSEquationFit ) ) {
+				} else if ( ( CoilType_Num == CoilDX_MultiSpeedCooling ) ) {
```
Comment (rraustad) [inline src/EnergyPlus/HVACUnitarySystem.cc]: Compiler warning of extraneous parentheses

**Comment by mjwitte** [comment]
@rraustad <sigh> More conflicts now.  If you can get those resolved, I will review tonight and try to get this in before any more merges cause conflicts.

**Comment by mjwitte** [comment]
Reviewed and tested, just waiting for CI to come back one more time.

**Comment by mjwitte** [comment]
@Myoldmopar @lefticus Not a big deal, but this branch doesn't show on the dashboard, probably because the branch name starts with a # ?

**Pull request merged**